test/02-mint.test.js

```diff
@@ -0,0 +1,59 @@
+const MyToken = artifacts.require("MyToken");
+const Assert = require('truffle-assertions');
+
+contract("Mint", accounts => {
+
+    let contractInstance;
+    const decimals = 6;
+    const ownerAddress = accounts[0];
+    const address1 = accounts[1];
+
+    beforeEach(async () => {
+        contractInstance = await MyToken.new();
+    });
+
+    it('mint should throw if contract is paused', async () => {
+        const mintValue = 1000;
+        
+        await contractInstance.pause({ from: ownerAddress });
+        
+        await Assert.reverts(
+            contractInstance.mint(address1, mintValue, { from: ownerAddress }),
+            'Pausable: paused'
+        );
```

Comment:
use openzeppelin test helper to track errors and so on.
Look in the channel for the internship, there are examples from ivan.bodnar or call and I'll explain everything to you